apps/ledger-live-desktop/src/renderer/screens/earn/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import Card from "~/renderer/components/Box/Card";
+import { languageSelector } from "~/renderer/reducers/settings";
+import { useRemoteLiveAppManifest } from "@ledgerhq/live-common/platform/providers/RemoteLiveAppProvider/index";
+import WebPlatformPlayer from "~/renderer/components/WebPlatformPlayer";
+import useTheme from "~/renderer/hooks/useTheme";
+import { useLocalLiveAppManifest } from "@ledgerhq/live-common/platform/providers/LocalLiveAppProvider/index";
+
+const DEFAULT_EARN_APP_ID = "earn";
+
+const Earn = () => {
+  const locale = useSelector(languageSelector);
+  const localManifest = useLocalLiveAppManifest(DEFAULT_EARN_APP_ID);
+  const remoteManifest = useRemoteLiveAppManifest(DEFAULT_EARN_APP_ID);
+  const manifest = localManifest || remoteManifest;
+  const themeType = useTheme("colors.palette.type");
+
+  return (
+    // TODO: Remove @ts-ignore after Card component be compatible with TS
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    <Card grow style={{ overflow: "hidden" }}>
+      {manifest ? (
+        <WebPlatformPlayer
+          config={{
+            topBarConfig: {
+              shouldDisplayName: false,
+              shouldDisplayInfo: false,
+              shouldDisplayClose: false,
+              shouldDisplayNavigation: false,
+            },
+          }}
+          manifest={manifest}
+          inputs={{
+            theme: themeType,
+            lang: locale,
+          }}
+        />
+      ) : null}
```

Comment:
What gets displayed if there is no manifest - just a blank screen? @cjordan-ledger

Reply:
I m to tired and black white 😅

Reply:
Sorry thanks

Reply:
Yes, the same behaviour as Buy / Sell Live App.